docs/package.json

```diff
@@ -16,7 +16,7 @@
   "dependencies": {
     "css-loader": "^0.23.0",
     "highlight.js": "^8.9.1",
-    "history": "^1.11.1",
+    "history": "1.13.0",
```

Comment:
this was addressed earlier, you can remove this change if you don't wanna deal with conflict resolution.